src/datasets/other/activity-stream/reference.md

```diff
@@ -1,132 +1,121 @@
+This data is measured in various custom pings that are sent via [PingCentre](https://searchfox.org/mozilla-central/source/browser/modules/PingCentre.jsm) (different from [Pingsender](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/internals/pingsender.html)).
 
-* [Activity Stream Pings: `data_events.md`](https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/data_events.md)
-* [Activity Stream Pings: `data_dictionary.md`](https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/data_dictionary.md)
-* [Activity Stream Pings: Firefox Source Tree Documentation](https://firefox-source-docs.mozilla.org/browser/components/newtab/docs/v2-system-addon/data_dictionary.html)
+* [Activity Stream Pings: `data_events.md`](https://firefox-source-docs.mozilla.org/browser/components/newtab/docs/v2-system-addon/data_events.html)
+* [Activity Stream Pings: `data_dictionary.md`](https://firefox-source-docs.mozilla.org/browser/components/newtab/docs/v2-system-addon/data_dictionary.html)
 
 ## Accessing Activity Stream Data
 
-The various Activity Stream pings are stored in tables stored in the `Tiles` Redshift database, maintained by the Activity Stream team. 
+Activity Stream pings are stored in the BigQuery database like other metrics of Firefox. It has two namespaces: `activity_stream` and `messaging_system`.
 
-This database can be accessed via [re:dash](https://sql.telemetry.mozilla.org/), or in Databricks via a [workaround provided by the Data Operations team](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/155100/command/155102), tracked in this [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1272388#c16). 
+#### Namespace `activity_stream`
 
+* Table `events` stores user interactions on the `about:home` and `about:newtab` pages
+* Table `sessions` stores sessions of `about:home` and `about:newtab` pages
+* Table `impression_stats` stores impression/click/block events for the Pocket recommendations on the `about:home` and `about:newtab` pages
+* Table `spoc_fills` stores Pocket Sponsored recommendation related pings
 
+#### Namespace `messaging_system`
+
+* Table `cfr` stores user interactions of the CFR (Contextual Feature Recommendation)
+* Table `moments` stores Moments Pages related pings
+* Table `onboarding` stores user interactions around the Onboarding features
+* Table `snippets` stores impression/click/dismissal of Firefox Snippets
+* Table `whats_new_panel` stores What's New Panel related pings
+* Table `undesired_events` stores system health related events
 
 ## Gotchas and Caveats
 
-Since this data collection isn't collected or maintained through our standard Telemetry API, there are a number of "gotchas" to keep in mind when working on this data. 
+Since this data collection isn't collected or maintained through our standard Telemetry API, there are a number of "gotchas" to keep in mind when working on this data.
```

Comment:
```suggestion
Since this data collection isn't collected or maintained through our standard Telemetry API, there are a number of "gotchas" to keep in mind when working on this data:
```